app/views/exchanges/scheduled_events/_scheduled_events_list.html.erb

```diff
@@ -0,0 +1,9 @@
+<ul>
+  <% if scheduled_events.present? %>
```

Comment:
I don't think this if statement should be necessary - .all should return an empty array